### PR TITLE
OBJECTS-1239: Move Simple Service guide/tutorial to here

### PR DESCRIPTION
**Pull request by ghost**
### What changes were proposed in this pull request?

Moves documentation on the simple service example and archetype to this repo.

#### Depends On

Nothing

### CODE REVIEW

File: guides/README.adoc [hunk 1]
```diff
@@ -46,6 +46,9 @@ link:ec2instance/README.adoc[AWS EC2 Instance Setup]::
 This guide leads you through the process of getting the SMART COSMOS DevKit
 running on an AWS EC2 Instance.
 
+link:simple-service/README.adoc[]::
```
Comment (andreassiegel) [inline guides/README.adoc]: Isn't there a link text missing?

**Review by tcrossrfid — COMMENTED** [review]
LGTM, after Andreas issue.

File: guides/README.adoc (hunk 1 above)
Comment (wfschneider-rfid) [inline guides/README.adoc]: yes there should be a title here

✅

**Review by andreassiegel — APPROVED** [review]

**Comment by ghostdock** [comment]
only have two thoughts
- Should we give on sentence to the Archetyp itself? "With the archtyp you are able.." or something like this?
- The New Project steps - .. one sentence that explain before why to have this? .."You will need to .." and/or may a links to docker itself and the installation guide regarding deployment?

**Pull request merged**